youtube_dl/extractor/vimeo.py

```diff
@@ -285,8 +285,14 @@ def _real_extract(self, url):
         # Extract the config JSON
         try:
             try:
-                config_url = self._html_search_regex(
-                    r' data-config-url="(.+?)"', webpage, 'config URL')
+                try:
+                    config_url = self._html_search_regex(
+                        r' data-config-url="(.+?)"', webpage, 'config URL')
+                except RegexNotFoundError:
```

Comment:
Instead of try-except you should pass `default=None` for first `config URL` and try alternative way when `not config_url`.